.github/workflows/build-test-docker.yaml

```diff
@@ -17,14 +17,21 @@ on:
         required: true
         type: string
         description: The repository/name of the docker image to push, e.g., returntocorp/semgrep
+        default: returntocorp/semgrep
       file:
         required: true
         type: string
         description: Dockerfile to build
+        default: Dockerfile
       enable-tests:
         required: true
         type: boolean
         description: Whether or not to run validation on the built image
+      push-immediately:
+        required: true
+        type: boolean
+        description: Whether or not to push immediately
```

Comment:
nit: this description doesn't really explain what this means, can you add an example or clarify in some other way?